packages/plugin-char/Char.ts

```diff
@@ -61,14 +62,12 @@ export class Char extends JWPlugin {
             const fragment = document.createDocumentFragment();
             let parent: Node = fragment;
             const renderedFormats = [];
-            Object.keys(node.format).forEach(type => {
-                if (node.format[type]) {
-                    const formatNode = document.createElement(Format.toTag(type));
-                    renderedFormats.push(formatNode);
-                    parent.appendChild(formatNode);
-                    // Update the parent so the text is inside the format node.
-                    parent = formatNode;
-                }
+            Object.values(node.format || {}).forEach(value => {
```

Comment:
I still don't understand why we have to handle the case where some property might not be defined. Just define them in the class once and be done with it.